interface/resources/qml/hifi/avatarapp/AvatarsModel.qml

```diff
@@ -20,18 +20,48 @@ ListModel {
             .split('%marketId%').join(extractMarketId(avatarUrl));
 
         return marketItemUrl;
+    }
+	
+	function makeMarketThumbnailUrl(marketId) {
+        var avatarThumbnailUrl = "https://hifi-metaverse.s3-us-west-1.amazonaws.com/marketplace/previews/%marketId%/large/hifi-mp-%marketId%.jpg"
+            .split('%marketId%').join(marketId);
```

Comment:
We shouldn't be using the name "hifi", and also is that an official hifi repo? We can expect those to go away eventually.

Also, is .split.join idiomatic JS?

Reply:
I think that this PR only works with the marketplace avatars from hifi at the moment. It probably won't work for long like this, but I think it's acceptable for now.

Reply:
split and join is part of the JS language yes.

Reply:
Yes before thumbnails only came from the marketplace. Just making sure do not break that functionality if you still have marketplace avatars bookmarked.